### ISSUE TITLE
Setting to turn off lightning bolt

### ISSUE THREAD

**Issue by albingroen** [zed-industries/feedback#443]
### Check for existing issues

- [X] Completed

### Is your feature request related to a problem?

I'm not keen on having lightning bolts everywhere in my code.

### Describe the solution you'd like

A setting to turn off the lightning bolts on the start of lines.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="620" alt="image" src="https://user-images.githubusercontent.com/19674362/184965837-71f0e232-3f70-430f-9219-3ba8e635ce13.png">

**Comment by JosephTLyons** [comment]
The lightning bolt is used to signify that a code action can be ran (`cmd-.` to trigger).  Disabling the lightning bolt would mean the user wouldn't know if there is a code action available or not - is that something you'd be ok with?  Would there be a better way to show that an action is available, or would you simply not care that you wouldn't know?  Just curious to hear your opinions.

**Comment by albingroen** [comment]
My workflow in my current editor (Neovim) is that I see a squiggly line under my code, and then I hit cmd+. to see the code actions. Isn't this flow pretty standard?

**Comment by JosephTLyons** [comment]
VS Code and JetBrains' IDEs both produce a lightbulb near the gutter - I'm guessing that might've been the inspiration behind the lightning bolt in Zed

<img width="266" alt="image" src="https://user-images.githubusercontent.com/19867440/184977340-a2ce7080-29d6-4da5-86bd-73820b1fe179.png">

**Comment by maxbrunsfeld** [comment]
> I see a squiggly line under my code, and then I hit cmd+. to see the code actions

In Zed, a squiggly line is already used to indicate an error or a warning, similar to VS Code or JetBrains IDEs. I would have thought that Neovim would use them that way too. The two concepts are related (because sometimes there are code actions for fixing an error), but not the same. Does the squiggly line actually indicate code actions in your Neovim setup?

**Comment by albingroen** [comment]
Zed
<img width="684" alt="image" src="https://user-images.githubusercontent.com/19674362/185035307-70b393ee-ba29-44c6-9ad4-de5ab3546fc1.png">

Neovim
<img width="731" alt="image" src="https://user-images.githubusercontent.com/19674362/185035330-cbec14c0-8905-48b3-b7e6-33f63cb85524.png">

**Comment by maxbrunsfeld** [comment]
Ok, it looks like they use a ‘>>’ instead of a lightning bolt?

**Comment by ksandvik** [comment]
Lightning bolt is fine.

**Comment by albingroen** [comment]
> Ok, it looks like they use a ‘>>’ instead of a lightning bolt?

Yes, but only on lines where code actions on available. In zed it's basically no every line. That's my issue here

**Comment by danieltroger** [comment]
> VS Code and JetBrains' IDEs both produce a lightbulb, near the gutter, when an action is available

I hate that and got annoyed by the lightning bolt too.
I think you should have a drawer where you list issues in the code and also have buttons for quick actions.

**Comment by peterkos** [comment]
To chime in, I personally don't use code actions in my workflow often -- only for expanding match arms. Otherwise I resort to a linter :) so I don't quite have a use for the lightning bolt, and don't mind disabling it if it was an option

**Comment by vaidotasp** [comment]
+1 on this issue, most of the files in codebase have code action available ("organize imports") which I never want to do, but because code action is available, that bolt is following my cursor on each file on each line. I can be very distracting.
